go.mod

```diff
@@ -1,13 +1,17 @@
 module github.com/kubernetes-csi/csi-proxy/v2
 
-go 1.22
+go 1.24.3
+
+godebug winsymlink=0
```

Comment:
Would this setting be applied transitively by CSI Drivers that import the CSI Proxy library? I think it's worth checking that with your CSI Driver given that the command to build the go binary would be in that codebase and not on this one.

Reply:
One long term goal is to address https://github.com/kubernetes-csi/csi-proxy/issues/374 where we don't need the `godebug ...` rule anymore given that it's a hack to make existing code work with go 1.23+. We should ideally remove this line while ensuring that there's no regression on CSI Drivers that use this go module.

Reply:
check the details here: https://github.com/kubernetes/kubernetes/pull/129368, it only impacts symbolic link for disk